doc/user/content/ingest-data/sql-server/self-hosted.md

```diff
@@ -70,6 +70,123 @@ Select the option that works best for you.
 
 {{< /tab >}}
 
+{{< tab "Use AWS PrivateLink">}}
```

Comment:
this is basically the same as the postgres version. do we want to create a shared template? or are we okay with just doing the doc maintenance per source?

I can't imagine it will change frequently or that documentation will become significant overhead in Source creation.